packages/batch/jest.config.cjs

```diff
@@ -5,6 +5,9 @@ module.exports = {
   },
   runner: 'groups',
   preset: 'ts-jest',
+  moduleNameMapper: {
+    '^(\\.{1,2}/.*)\\.js$': '$1',
+  },
```

Comment:
This is addition to all the jest config files is needed because we are now importing packages via relative path making the extension explicit.

This is a Typescript requirement and not a style choice, read [here for more info](https://www.totaltypescript.com/relative-import-paths-need-explicit-file-extensions-in-ecmascript-imports) on why.

The new `moduleNameMapper` key tells Jest how to interpret these new paths, and it tells it to remove the `.js` extension. This is because Jest doesn't work with ESM at all so we are basically treating the tests as CJS. In the near/medium future we're going to probably move away from Jest entirely in favor of Vite, but for now this is how we make it work.